src/DevSoc.Eventathon/ClientApp/src/components/ConfirmAttendance.js

```diff
@@ -3,18 +3,19 @@ import axios from "axios";
 import {useHistory} from "react-router-dom";
 
 export const ConfirmAttendance = () => {
+    
+    // Retrieve parameters from the EventsCalender page
```

Comment:
these comments are unnecessary - it's clear what's happening here without the comment, and in the future we could have other pages that also redirect here (other than the `EventsCalendar`)

Reply:
@EvelynVoce has this been resolved?